utils/docker/Dockerfile.el.8

```diff
@@ -63,7 +65,11 @@ gpgcheck=False\n" >> /etc/yum.repos.d/$repo:$branch:$build_number.repo;   \
 
 # Install OS updates and package.  Include basic tools and daos dependencies
 COPY ./utils/scripts/install-el8.sh /tmp/install.sh
-RUN chmod +x /tmp/install.sh && dnf upgrade && /tmp/install.sh && dnf clean all && \
+RUN chmod +x /tmp/install.sh && dnf upgrade &&           \
+    if [ -n "$REPO_FILE_URL" ]; then                     \
+        dnf config-manager --disable epel-\* ;      \
+    fi &&                                                \
+    /tmp/install.sh && dnf clean all &&                  \
```

Comment:
I'm still not sure I fully get this change, is the issue that the epel repo includes a newer version of the epel-release package so if you update after installing epel-release it reconfigures dnf to use the real repo rather than our mirror?

If that's the case I wonder if setting excludepkgs=epel-release for the epel repo would work, and would be something that could be configured once rather than on every update.

Reply:
We want the current epel-release package to be installed.   We just do not want the side effect of it re-enabling the distro repo if we previously disabled it.

The new epel-release package can install updated GPG keys.